src/components/pages/ProfilePage/types.ts

```diff
@@ -33,11 +33,55 @@ export type PopoverT = PropsWithChildren<{
   buttonStyle?: CSSProperties;
   buttonContent?: ReactNode;
   overlayStyle?: CSSProperties;
+  getVideoLink?: () => void;
 }>;
 export type ContentButtonT = {
   icon: string;
   text: string;
+  getVideoLink?: () => void | undefined;
```

Comment:
```suggestion
  getVideoLink?: () => void;
```
I believe we don't need `undefined` here?